main.go

```diff
@@ -27,34 +27,37 @@ import (
 	"github.com/gardener/etcd-wrapper/internal/bootstrap"
 	"github.com/gardener/etcd-wrapper/internal/signal"
 	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
 )
 
-const (
-	defaultLogLevel = zapcore.InfoLevel
-)
+// const (
+// 	defaultLogLevel = zapcore.InfoLevel
+// )
 
 func main() {
 	args := os.Args[1:]
 	checkArgs(args)
-	logger, err := bootstrap.SetupLogger(defaultLogLevel)
+
+	//create logger
+	loggerCfg := bootstrap.SetupLoggerConfig(types.DefaultLogLevel)
+	mainLogger, err := loggerCfg.Build()
```

Comment:
Why is this renamed to `mainLogger`? There's only one logger used in this function, so using the name `logger` seems fine to me.

Reply:
renamed it to `logger`